render.sh

```diff
@@ -1,68 +1,180 @@
 #!/usr/bin/env bash
 set -e
 
-ORG=${1:-""}
-REPO=${2:-""}
-BRANCH=${3:-"ptc/data"}
-DIR=${4:-""}
-FORECASTS=${5:-""}
-EVALS=${6:-""}
-if [[ $ORG == "hubverse-org" && $REPO == "hub-dashboard-predtimechart" ]]; then
-  DIR="demo/"
-fi
-if [[ -z $ORG && -z $REPO ]]; then
-  # the default will be the repo from the config
-  full=$(yq .hub /site/site-config.yml)
-  ORG=${full%%/*} # bash expansion: delete longest match after '/'
-  REPO=${full#*/} # bash expansion: delete shortest match before '/'
-  REPO=${REPO%/*} # bash expansion: delete shortest match after '/'
+PTC=""
+PREDEVALS=""
+
+render_help () {
+  echo "Render a dashboard website"
+  echo
+  echo "USAGE"
+  echo
+  echo "  render.sh [-h] [-u <user> -r <repo>] [-p <ptc/data> -e <predevals/data>] [-o <outpath>]"
+  echo
+  echo "ARGUMENTS"
+  echo
+  echo "  -h  print help and exit"
```

Comment:
I have this a spin, and it did indeed print the help and exit...it closed the terminal window before I could read the help.

I don't have a lot of experience writing CLIs in bash, so I'm not if there's a fix for this or no.

Reply:
> it closed the terminal window before I could read the help.

OH WOW. I thought `exit 0` would just exit the current script, not close the dang window.

Reply:
I think I know why.... did you happen to use `. render.sh` or `source render.sh`? 

This script is intended to be an executable, so it should be run with `bash render.sh` or just `./render.sh` (https://stackoverflow.com/q/9640660)

Reply:
Yep, I was doing `source render.sh`. Switching to `bash render.sh` resulted in the expected behavior!

Reply:
I added examples.md for help with this. That being said, the only way to test this is to actually build the container and then run the script from within.

Reply:
I've added a test script and instructions in the README, but see https://github.com/hubverse-org/hub-dash-site-builder/pull/21#issuecomment-2758660836